mysql/datadog_checks/mysql/statements.py

```diff
@@ -150,6 +157,18 @@ def collect_per_statement_metrics(self):
             'mysql_rows': rows,
         }
         self._check.database_monitoring_query_metrics(json.dumps(payload, default=default_json_event_encoding))
+        self._check.histogram(
```

Comment:
for this time metric, should we instead just instrument the method with APM like we do the sqlserver integration? https://github.com/DataDog/integrations-core/blob/master/sqlserver/datadog_checks/sqlserver/statements.py#L355

Reply:
sure!